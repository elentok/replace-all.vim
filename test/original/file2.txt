I am jim2
I am joe2
I am bob2
I am bob8
