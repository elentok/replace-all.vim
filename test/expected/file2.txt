I am jim2
I am joe2
I am /*#sheldon"2
I am &sheldon
